src/ui/Card.tsx

```diff
@@ -189,7 +189,11 @@ export function Card(props: CardProps): JSX.Element {
     },
   );
 
-  const [angleOffset] = useState(randomSign() * Math.random());
+  // Don't rotate the front card.
```

Comment:
I think I like this conceptually, but using the preventRenderMessages prop is clearly a hack.

Does passing in an additional "isFrontCard" prop make sense?

Do you like this conceptually?

Reply:
Using preventRenderMessages is wrong actually since we render the first *two* cards' messages. So, adding an isFrontCard prop makes sense to me.